contracts/bid/PartyBid.sol

```diff
@@ -38,6 +38,9 @@ import {
     IERC721Metadata
 } from "@openzeppelin/contracts/token/ERC721/extensions/IERC721Metadata.sol";
 import {IWETH} from "../external/interfaces/IWETH.sol";
+import {
```

Comment:
Can we update the version number since this is a new ABI?

Reply:
this will be included in the PartyBuy release, version number is updated in that PR